includes/definitions/allied-wireless.yaml

```diff
@@ -0,0 +1,14 @@
+os: allied-wireless
```

Comment:
A look at https://www.alliedtelesis.com/sites/default/files/tq4600_tq4400e_4.0.3_srn_rev_a.pdf indicates this might be better as `allied-tq`?

Reply:
Hmm quite possibly. 

I guess I was just trying to start something off that's generic, but perhaps being more specific is the way to go, given the wireless range I summarize below. 

To summarize:
- TQ APs are the "premium" access points which Allied Telesis make. 
- There's now also the MWS range which I think is lower in price/performance (and does have different mibs)
- There's also the Extricom wireless range (from when AT bought Extricom) which I believe is EOL.
- Then there's the AW+ Wireless controller with some form of different mibs support, which can manage the TQ APs. 

Probably some other range I'm forgetting about....

I'll change it to allied-tq.